src/lib/ethereum/tokens.ts

```diff
@@ -18,6 +18,8 @@ export enum Token {
     LUNA = "LUNA",
     DOGE = "DOGE",
     DGB = "DGB",
+    renBTC = "renBTC",
+    FTM = "FTM",
```

Comment:
```suggestion
    FTM = "FTM",

    // Fee tokens (community holdings)
    renBTC = "renBTC",
```

Reply:
pushed a new commit to remove all any type